docs/dsp-tools-usage.md

```diff
@@ -92,60 +104,55 @@ dsp-tools xmlupload [options] xml_data_file.xml
 The following options are available:
 
 - `-s` | `--server` (optional, default: `0.0.0.0:3333`): URL of the DSP server 
-- `-u` | `--user` (optional, default: `root@example.com`): username used for authentication with the DSP API 
-- `-p` | `--password` (optional, default: `test`): password used for authentication with the DSP API
-- `-i` | `--imgdir` (optional, default: `.`): path to the directory where the bitstream objects are stored
+- `-u` | `--user` (optional, default: `root@example.com`): username used for authentication with the DSP-API 
+- `-p` | `--password` (optional, default: `test`): password used for authentication with the DSP-API
+- `-i` | `--imgdir` (optional, default: `.`): folder where the paths in the `<bitstream>` tags are evaluated from
```

Comment:
Why "evaluated" only? I think the images can also be taken from there. Or am I wrong?

Reply:
What I mean is: the `<bitstream>` tag contains relative pathes. DSP-TOOLS computes these relative pathes from the `--imgdir`. That means, the images are taken from the folder that results from imgdir + relative path.

It's a big complicated to explain, I don't have the right words...